docs/content/contributing/terraform/review.md

```diff
@@ -0,0 +1,75 @@
+---
+title: AVM Review of Terraform Modules
+geekdocNav: true
+geekdocAlign: left
+geekdocAnchor: true
+---
+
+{{< toc >}}
+
+AVM Review is a critical step before an AVM terraform module is published to [Terraform Registry](https://registry.terraform.io/) and made publicly available for customers, partners and wider IaC community to consume and contribute to. It acts as a quality assurance step to ensure that the AVM terraform module complies with the [shared](https://azure.github.io/Azure-Verified-Modules/specs/shared/) and [terraform](https://azure.github.io/Azure-Verified-Modules/specs/terraform/) specifications of AVM. The below process outlines the steps for both a module owner and a module reviewer should follow to do an AVM Review of a terraform module.
+
+1. The module owner completes the development of the module in their branch or fork.
+
+2. The module owner does a pull request (PR) titled 'AVM-Review-PR' ensuring that all checks are passing on that PR as that is a pre-requisite to request a review.
+
+3. The module owner assigns the `avm-core-team-technical-terraform` as reviewers on the PR.
+
+4. The module owner leaves the following comment as is on the module proposal in the AVM Repo by searching for their module proposal here.
+<br>"I would like to request a review of my module. The latest code is in a PR titled AVM-Review-PR on the module repo and all checks on that PR are passing."
+
+5. The AVM team will move the module proposal from "In Development" to "In Review" in the [AVM - Module Triage](https://github.com/orgs/Azure/projects/529) project.
+
+6. The AVM team will assign a module reviewer who will open a blank issue on the module titled "AVM-Review" and populate it with the below mark down. This template already marks the specs as compliant which are covered by the checks that run on the PR. There are some specs which don't need to be checked at the time of publishing the module therefore they are marked as NA.
```

Comment:
@mbilalamjad, I really like this flow, but why don't we add this as an issue template through the template repo? It could just be called "Initial module review" - this way people won't have to copy text around.

Reply:
That's a good idea but worried that it might confuse module owners and module consumers as to when to create an issue with that template